pages/thc/[processRef]/pause.tsx

```diff
@@ -43,9 +61,12 @@ const PausePage: NextPage = () => {
             : "You are currently working offline."}
         </Paragraph>
         <Paragraph>
-          The Tenancy and Household Check for the tenancy at {address}, occupied
-          by {tenants.join(", ")} has been saved to your device but still needs
-          to be saved to your work tray so you can resume it later.
+          The Tenancy and Household Check for the tenancy at
+          <React.Fragment key="address">{addressLab}</React.Fragment> occupied
+          by
+          <React.Fragment key="tenant">{tenantLab}</React.Fragment>has been
+          saved to your device but still needs to be saved to your work tray so
+          you can resume it later.
```

Comment:
```suggestion
          The Tenancy and Household Check for the tenancy at {addressLab} occupied by {tenantLab} has been
          saved to your device but still needs to be saved to your work tray so you can resume it later.
```

No need to use the fragments here.